snare.py

```diff
@@ -178,6 +179,10 @@ def handle_request(self, request, payload):
         # Submit the event to the TANNER service
         event_result = yield from self.submit_data(data)
 
+        #Log the URL to /opt/snare/snare.log
+        ip=self.transport.get_extra_info('peername')[0]
+        self.logger.info('Source IP %s - - Requested path %s', ip, self.run_args.page_dir + request.path)
```

Comment:
Please use the Apache access log format: https://httpd.apache.org/docs/current/logs.html#accesslog